packages/core/src/scripts/telemetry.ts

```diff
@@ -0,0 +1,121 @@
+import chalk from 'chalk';
+import Conf from 'conf';
+import { Configuration } from '../types/telemetry';
+
+export async function telemetry(cwd: string, option?: string) {
+  const usageText = `
+  The telemetry command requires a valid option
+  
+      Usage
+        $ keystone telemetry [option]
+      Options
+        status      displays the current telemetry configuration
+        reset       resets the current telemetry configuration (if any)
+        enable      enables telemetry for the current user
+        disable     resets the current telemetry configuration (if any) and disables all telemetry for the current user
+      `;
+
+  const helpText = `
+  ${chalk.bold('KeystoneJS Telemetry')}
+
+      Usage
+        $ keystone telemetry [option]
+      Options
+        status      displays the current telemetry configuration
+        reset       resets the current telemetry configuration (if any)
+        enable      enables telemetry for the current user
+        disable     resets the current telemetry configuration (if any) and disables all telemetry for the current user
+  
+  For more details visit: https://keystonejs.com/telemetry    
+        `;
+
+  const disabledText = `
+KeystoneJS telemetry: ${chalk.red('Disabled')}
+    
+  Keystone telemetry is disabled on this device.
+  For more details visit: https://keystonejs.com/telemetry`;
+
+  const enabledText = (telemetryData: Configuration['telemetry']) => `
+KeystoneJS telemetry: ${chalk.green('Enabled')}
+   
+  Telemetry is configured as follows:
+
+${JSON.stringify(telemetryData, null, 2)}
+
+  Telemetry is completely anonymous and helps us make Keystone better.
+  For more details visit: https://keystonejs.com/telemetry`;
+
+  const setupText = `
+KeystoneJS telemetry: ${chalk.red('Not Inilialized')}
+
+  Please run ${chalk.green(
+    'keystone telemetry setup'
```

Comment:
`keystone telemetry setup` doesn't exist right?

Reply:
That's correct, this has been fixed now